apps/ts-minbar-test-react-components/assets/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "jsx": "react",
+    "lib": ["es2015", "dom"],
+    "target": "es5",
```

Comment:
These can probably use ES2019

Reply:
Also I mentioned in chat that you probably need `moduleResolution: node` but based on the docs (https://www.typescriptlang.org/tsconfig#moduleResolution) it may default to that if you switch the target to ES2019.